runtimes/standalone/src/lib.rs

```diff
@@ -400,29 +404,29 @@ construct_runtime!(
 		NodeBlock = opaque::Block,
 		UncheckedExtrinsic = UncheckedExtrinsic
 	{
-		System: frame_system::{Module, Call, Config, Storage, Event<T>},
-		RandomnessCollectiveFlip: randomness_collective_flip::{Module, Call, Storage},
+		System: frame_system::{Module, Call, Config, Storage, Event<T>} = 0,
+		RandomnessCollectiveFlip: randomness_collective_flip::{Module, Call, Storage} = 1,
```

Comment:
Is there a reason for the explicit indices?

Reply:
I saw this in the [Polkadot runtimes](https://github.com/paritytech/polkadot/blob/master/runtime/polkadot/src/lib.rs#L983) and liked it because we are actually using the indices for the extrinsic errors in the SDK. It. should have definitely been a separate PR so I am happy to revert it and propose it later.

Reply:
na let's keep it. I reviewed it already! 😁 Another PR means that I have to review it AGAIN!!! 😨